client/controllers/events/eventAffectedAreas.coffee

```diff
@@ -153,7 +153,8 @@ Template.eventAffectedAreas.onRendered ->
     EventIncidents.find(@data.filterQuery.get()).map (incident) ->
       incident.locations.forEach (location) ->
         key = "#{location.latitude},#{location.longitude}"
-        incidentsByLatLng[key] = (incidentsByLatLng[key] or []).concat(incident)
+        if not incidentsByLatLng[key]
```

Comment:
@jgoley The problem was that the same incident could be added to the same coordinate twice if it has two locations with the same coordinate. I added a commit that fixes this by deduplicating the incidents for each coordinate.